packages/odyssey-react-mui/src/labs/DataComponents/TableSettings.tsx

```diff
@@ -37,24 +38,17 @@ const TableSettings = ({
   const { rowDensity, columnVisibility } = tableState;
 
   const changeRowDensity = useCallback(
-    (value: (typeof densityValues)[number]) =>
-      (_event: React.MouseEvent<HTMLLIElement>) => {
-        // This is necessary to avoid linter errors, while the _event is necessary to satisfy the onClick type
-        if (process.env.NODE_ENV === "development") console.debug(_event);
-
-        setTableState((prevState) => ({
-          ...prevState,
-          rowDensity: value,
-        }));
-      },
+    (value: MRT_DensityState) => {
+      setTableState((prevState) => ({
+        ...prevState,
+        rowDensity: value,
+      }));
+    },
     [setTableState],
   );
 
   const changeColumnVisibility = useCallback(
```

Comment:
@jordankoschei-okta I think we started moving towards defining these arguments in the generic we can pass to `useCallback`
```suggestion
  const changeColumnVisibility = useCallback<(columnId: string) => void>(
```
Rather then typing the arg in the function